src/ui/load.js

```diff
@@ -37,7 +42,18 @@ export async function load(): Promise<LoadResult> {
     const rawLedger = await responses[2].text();
     const ledger = Ledger.parse(rawLedger);
     const {hasBackend} = await responses[3].json();
-    return {type: "SUCCESS", credView, bundledPlugins, ledger, hasBackend};
+    const {
+      currencySuffix: suffix = DEFAULT_SUFFIX,
+      currencyName: name = "Grain",
+    } = grainConfigParser.parseOrThrow(await responses[4].json());
```

Comment:
I'd prefer to have the defaults live at the level of the config file, e.g. we could have a type `SerializedGrainConfig` which has the fields as optional, and then a `GrainConfig` has the fields as required, and we'd have some sort of `upgrade(g: SerializedGrainConfig): GrainConfig` function which applies the defaults. Then the parser would compose a `SerilaizedGrainConfig` parse step along with using `fmap` to apply the upgrade. That way, we can ensure that the same defaults are used consistently across the system, v.s. a case where we might call the default `"Grain"` here and `"grain"` elsewhere and have slight inconsistencies. 

Also, now that I'm thinking about it more, we may want to have a config file which is only focused on the currency display stuff, e.g. call it `config/currencyDetails.json`. Looking at `config/grain.json`, to have it both give currency details and payout strategy feels like a messy mixing of concerns... the payout strategy and the rendering info are actually quite distinct. What do you think? Would merge it either way, but I think it may be a better design to separate them.

Reply:
My instinct was to create a separate `currencyDetails.json` file (I need to start putting these thoughts in the PR descriptions).  Combining data used for completely different applications indeed seems inconsistent with the design philosophy of the project, so I agree we should fix this here. I'm excited to dig deeper into Combo functionality as well!